Build error in Fedora 25

#### Additional Information
(The following information is very important in order to help us to help you. Omission of the following details may delay your support request or receive no attention at all.)

- Version of K2HASH being used (k2hlinetool -libversion)
 ```
K2HASH library Version 1.0.52 (commit: bcd8209)

Copyright 2013 Yahoo! JAPAN corporation.

K2HASH is key-valuew store base libraries. K2HASH is made for
the purpose of the construction of original KVS system and the
offer of the library. The characteristic is this KVS library
which Key can layer. And can support multi-processing and
multi-thread, and is provided safely as available KVS.


FULLOCK Version 1.0.21 (commit: 9157151)

Copyright 2015 Yahoo! JAPAN corporation.

FULLOCK is fast locking library on user level by Yahoo! JAPAN.
FULLOCK is following specifications.
 ```

- System information (uname -a)
 ```
Linux f844098a57bd 4.6.6-200.fc23.x86_64 #1 SMP Wed Aug 10 23:13:35 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
 ```

> Fedora 25 docker container on Fedora 23

- Distro (cat /etc/issue)
 ```
\S
Kernel \r on an \m (\l)

 ```

#### Details about issue
Build error in Fedora 25

> Fedora 25 / gcc version 6.2.1 20160916 (Red Hat 6.2.1-2) (GCC)

```
k2hmmapinfo.cc: In member function 'k2h_mmap_info** K2HMmapMan::GetMmapInfo(const K2HShm*, const char*, bool)':
k2hmmapinfo.cc:351:10: error: cannot convert 'bool' to 'k2h_mmap_info**' in return
   return false;

k2htouch.cc: In function 'bool AttrSet(K2HShm*)':
k2htouch.cc:291:26: error: cannot convert 'bool' to 'const bool*' in initialization
  const bool* is_defenc = false ;
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -29,7 +29,6 @@ before_install:
  - sudo apt-get update -qq
  - sudo apt-get install -qq git autoconf autotools-dev gcc g++ make gdb dh-make fakeroot dpkg-dev devscripts libtool pkg-config
  - git submodule update --init --recursive
- - git clone --recursive git://github.com/nakatani-yj/nakatani_yj_test_pj
  - cd fullock
  - ./autogen.sh
  - ./configure --prefix=/usr

--- a/lib/k2hmmapinfo.cc
+++ b/lib/k2hmmapinfo.cc
@@ -348,7 +348,7 @@ PK2HMMAPINFO* K2HMmapMan::GetMmapInfo(const K2HShm* pk2hshm, const char* file, b
 {
 	if(!pk2hshm){
 		ERR_K2HPRN("pk2hshm object pointer is NULL.");
-		return false;
+		return NULL;
 	}
 
 	if(needlock){

--- a/tests/k2hbench.cc
+++ b/tests/k2hbench.cc
@@ -467,7 +467,7 @@ static bool SetBenchOptions(int argc, char** argv, BOPTS& benchopts)
 			ERR("\"-ro\" option does not need parameter(%s).", optparams["-ro"].rawstring.c_str());
 			return false;
 		}
-		if(ISEMPTYSTR(benchopts.szfile) || benchopts.is_temp_mode){
+		if('\0' == benchopts.szfile[0] || benchopts.is_temp_mode){
 			ERR("\"-ro\" option could not set with \"-m\" or \"-t\" option.");
 			return false;
 		}
@@ -480,7 +480,7 @@ static bool SetBenchOptions(int argc, char** argv, BOPTS& benchopts)
 			ERR("\"-nofullmap\" option does not need parameter(%s).", optparams["-nofullmap"].rawstring.c_str());
 			return false;
 		}
-		if(ISEMPTYSTR(benchopts.szfile) || benchopts.is_temp_mode){
+		if('\0' == benchopts.szfile[0] || benchopts.is_temp_mode){
 			ERR("\"-nofullmap\" option could not set with \"-m\" or \"-t\" option.");
 			return false;
 		}
@@ -571,7 +571,7 @@ static bool SetBenchOptions(int argc, char** argv, BOPTS& benchopts)
 			ERR("\"-proc\" option parameter(%s) must be decimal string.", optparams["-proc"].rawstring.c_str());
 			return false;
 		}
-		if((ISEMPTYSTR(benchopts.szfile) || benchopts.is_temp_mode) && 1 < optparams["-proc"].num_value){
+		if(('\0' == benchopts.szfile[0] || benchopts.is_temp_mode) && 1 < optparams["-proc"].num_value){
 			ERR("\"-proc\" option parameter(%d) could not set with \"-t\" or \"-m\" option, must be \"-proc 1\" or not set.", optparams["-proc"].num_value);
 			return false;
 		}
@@ -670,7 +670,7 @@ static K2HShm* InitializeK2HashFile(BOPTS& benchopts)
 
 	// build file(memory)
 	bool	result;
-	if(ISEMPTYSTR(benchopts.szfile)){
+	if('\0' == benchopts.szfile){
 		result = pk2hshm->AttachMem(benchopts.maskcnt, benchopts.cmaskcnt, benchopts.elementcnt, benchopts.pagesize);
 	}else{
 		result = pk2hshm->Create(&benchopts.szfile[0], benchopts.is_fullmap, benchopts.maskcnt, benchopts.cmaskcnt, benchopts.elementcnt, benchopts.pagesize);
@@ -833,8 +833,8 @@ static void PrintResult(const PEXECCNTL pexeccntl, const PCHLDCNTL pchldcntl, st
 	PRN("===========================================================");
 	PRN("K2Hash bench mark");
 	PRN("-----------------------------------------------------------");
-	PRN("K2Hash type                   %s",			pexeccntl->opt.is_temp_mode ? "Temporary file" : ISEMPTYSTR(pexeccntl->opt.szfile) ? "Memory(no file)" : "Permanent file");
-	PRN("File path                     %s",			pexeccntl->opt.is_temp_mode ? "tmpfile on system" : ISEMPTYSTR(pexeccntl->opt.szfile) ? "no file" : pexeccntl->opt.szfile);
+	PRN("K2Hash type                   %s",			pexeccntl->opt.is_temp_mode ? "Temporary file" : ('\0' == pexeccntl->opt.szfile[0]) ? "Memory(no file)" : "Permanent file");
+	PRN("File path                     %s",			pexeccntl->opt.is_temp_mode ? "tmpfile on system" : ('\0' == pexeccntl->opt.szfile[0]) ? "no file" : pexeccntl->opt.szfile);
 	PRN("Attach type                   %s",			pexeccntl->opt.is_read_only ? "Read only" : "Read/Write");
 	PRN("Mapping type                  %s",			pexeccntl->opt.is_fullmap ? "Full mapping" : "Partial mapping");
 	PRN("Data structure");
@@ -1290,7 +1290,7 @@ int main(int argc, char** argv)
 	PrintResult(pexeccntl, pchldcntl, realtime);
 
 	// cleanup
-	if(!ISEMPTYSTR(benchopts.szfile)){
+	if('\0' != benchopts.szfile[0]){
 		unlink(benchopts.szfile);
 	}
 	munmap(pShmBase, totalsize);

--- a/tests/k2htouch.cc
+++ b/tests/k2htouch.cc
@@ -288,11 +288,10 @@ bool AttrSet(K2HShm* k2hash)
 	time_t *argExpire = NULL ;
 	if (expiretime) argExpire = &expiretime ;
 
-	const bool* is_defenc = false ; 
-	const char* passfile = "" ; 
+	const char*	passfile = "" ; 
 
 	return k2hash->SetCommonAttribute(
-		&is_mtime, is_defenc, passfile, &is_history, argExpire, pluginlibs) ;
+		&is_mtime, NULL, passfile, &is_history, argExpire, pluginlibs) ;
 }
 
 bool AttrClear(K2HShm* k2hash)

